source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -277,18 +277,40 @@ def wrapper(*args, **kwargs):
                         if not app_value:
                             continue
 
+                        # Get the territory value for the line used
+                        line_id = next(
+                            (link['id'] for link in s.get('links', []) if link['type'] == "line"), None
+                        )
+                        if line_id:
```

Comment:
```suggestion
                        territory_value = None
                        if line_id:
```
otherwise, in case `lind_id is None`,   [line 312](https://github.com/hove-io/navitia/pull/4068/files#diff-9f144f0945ab2aaeaed8f7771af1bfe0e9a14c86d0317959d37734ff6b0eecedR312) may crash

Reply:
:+1: